HIP/hip-0000-cross-ledger-protocol-clpr.md

```diff
@@ -0,0 +1,1152 @@
+---
+hip: <HIP number (assigned by the HIP editor), usually the PR number>
+title: Cross-Ledger Protocol (CLPR)
+author: Edward Wertz <@edward-swirldslabs>
+working-group: Leemon Baird <leemon@hashgraph.com>, Richard Bair <@rbair32>, Jasper Potts <@jasperpotts>, Joseph Sinclair <@jsync-swirlds>, Rohit Sinha <@rsinha>, Kiran Pachhai <@kpachhai>, Torfin Olsen <torfinn@hashgraph.com>, Edward Wertz <@edward-swirldslabs>
+requested-by: Leemon Baird <leemon@hashgraph.com>
+type: "Standards Track"
+category: "Core"
+needs-council-approval: "Yes"
+status: "Idea"
+created: 2026-06-18
+discussions-to:
+  <A URL pointing to the official discussion thread. Ex: https://github.com/hashgraph/hedera-improvement-proposal/discussions/000>
+updated: <Latest date HIP was updated, in YYYY-MM-DD format.>
+requires: 1200
+replaces: N/A
+superseded-by: N/A
+---
+
+## Abstract
+
+This HIP introduces the `Cross-Ledger PRotocol` or `CLPR`, pronounced `Clipper`
```

Comment:
Like the name much better do we need the `CLPR` at all any more so can we just say "called Clipper" rather than "pronounced Clipper"